### ISSUE TITLE
Layout of Media/Avatar edit screen

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/limelight#118]
currently the media edit screen is largely taken up by a full-size image of the media item, which makes it necessary (on my small screen) to scroll down in order to replace the file and access the Update button. presumably there is more to come on this page as well, e.g. voice options, download, etc.
there's no need to have a full-size image - it could be similar to UpStage v3 where there is a "preview" button that opens the full-size image in a pop-up window & allows you to play the animation if it's animated. 
mostly when we are working with media, we don't need to see a full-size image except on the stage. so a thumbnail image should suffice on the edit page.

**Comment by vickis** [comment]
is the thumbnail sizing to do with scarcity (will they decrease once more items are there?)
i am currently not having a size issue (but on 17" screen) 

is this for ease also on mobile devices?
preview of larger image definitely if decreasing thumbnail  size  :)

**Comment by rohrlach** [comment]
Yes - the image is responding to the fact that there's not much on the grid currently (I expect this to change) let's come back to this once we've got all the intended info there

**Comment by helenvarley** [comment]
great! thank you :)
i don't see the option to display as thumbnails anymore ... ?

**Comment by helenvarley** [comment]
actually what this issue is referring to is the media edit interface, e.g. https://limelight.upstage.org.nz/avatars/snoopy/edit, NOT the list of media. the media edit interface still has an enormous image (i guess it's full size) with the different tools (add/delete tags, delete icon, browse for replacing the file, & update) all at opposite corners of the screen. it's not a good use of space. in UpStage v3, the media edit screen fits things very neatly into a small area of the screen with a small preview image, AND you still have the list of the rest of the media visible on the left so that if this isn't the media item you wanted you can easily browse to find another.
i would recommend that this layout from UpStage v3 is used as the basis for the new media edit interface.
![screenshot at 2017-05-18 13 26 54](https://cloud.githubusercontent.com/assets/3437324/26200408/49096168-3bce-11e7-9b89-bb79e7901825.png)
![screenshot at 2017-05-18 13 29 23](https://cloud.githubusercontent.com/assets/3437324/26200407/49039aee-3bce-11e7-879c-3d8064ae7af2.png)
